### PR TITLE
Phuong created a simple visualization for MP2

### PR DESCRIPTION
**Pull request by pchau1807**
MP2 submission

### CODE REVIEW

File: 390/Phuong_Chau.Rmd [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# Load libraries
+```{r, echo = FALSE, warning=FALSE}
+library(readr)
+library(tidyverse)
+library(dplyr)
+```
+
+```{r, echo = FALSE, warning=FALSE}
+# Load data and remove NA values
+stem1 <- read_csv("C:/Fall2020_Lastsemester/EcologicalFocasting/SCBI-ForestGEO-Data/tree_main_census/data/census-csv-files/scbi.stem1.csv")
```
Comment (rudeboybert) [inline 390/Phuong_Chau.Rmd]: Hey Candy, here is a code review!

Unfortunately, this code doesn't work on my computer b/c of the filepath. Check out this comment I made on Lauren's PR that will set up code that will work on anyone's computer https://github.com/390-2020-09-Fall/SCBI-ForestGEO-Data/pull/2#discussion_r491179003

Comment (rudeboybert) [reply 390/Phuong_Chau.Rmd]: Could you fix this and push to this branch?

File: 390/Phuong_Chau.Rmd [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# First distribution
+hist(stem1$dbh, breaks=100, ylim = c(0,30000),col=rgb(1,0,0,0.5), xlab="dbh", 
+     ylab="frequency", main="distribution of dbh of trees measured in first 2 censuses" )
+
+# Second with add=T to plot on top
+hist(stem2$dbh, breaks=100, ylim = c(0,30000), col=rgb(0,0,1,0.5), add=T)
+
+# Add legend
+legend("topright", legend=c("Census 1: 2008-2010","Census 2: 2013"),
```
Comment (rudeboybert) [inline 390/Phuong_Chau.Rmd:39]: Looks good! Start thinking about how you can make a shiny app.

File: 390/Phuong_Chau.Rmd (hunk 1 above)
Comment (rudeboybert) [reply 390/Phuong_Chau.Rmd]: Once you fix this and the .Rmd file knits, I'll merge into `master`

Comment (pchau1807) [reply 390/Phuong_Chau.Rmd]: Hi Prof. Kim! I have changed the read_csv() and committed again but I don't see the "Compare and pull request" anymore. What should I do next after I committed?

Comment (rudeboybert) [reply 390/Phuong_Chau.Rmd]: Great question! When you make commits to a still unmerged pull request, the pull request auto-updates. In other words, you don't need to make a new one.

But if you make changes to a branch that's already been merged, then you will need to create a new PR.

**Comment by rudeboybert** [comment]
Looks good! Going to merge into `master`

**Pull request merged**